extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ amp-story-dev-tools-tab-preview {
+  --i-amphtml-story-dev-tools-device-background: #101010 !important;
+  --i-amphtml-story-dev-tools-device-screen: #2E3137 !important;
+  --i-amphtml-story-dev-tools-device-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+  --i-amphtml-story-dev-tools-device-chip-background: #2e2e2e !important;
+  --i-amphtml-story-dev-tools-device-chip-background-hover: #3d3d3d !important;
+  --i-amphtml-story-dev-tools-device-chip-icon-background: black !important;
+
+  --i-amphtml-story-dev-tools-device-dialog-background: #111111cc !important;
+  --i-amphtml-story-dev-tools-device-dialog-help-url-border: #ffffff22;
+  --i-amphtml-story-dev-tools-device-dialog-help-url-color: #ffffff77;
+ }
+
+.i-amphtml-story-dev-tools-device {
+  background: var(--i-amphtml-story-dev-tools-device-background) !important;
+  border-radius: 40px !important;
+  box-shadow: var(--i-amphtml-story-dev-tools-device-shadow) !important;
+  overflow: hidden !important;
+  position: absolute !important;
+  margin: auto !important;
+  height: fit-content !important;
+  top: 0 !important;
+  bottom: 0 !important;
+  padding: 10px 8px !important;
+  transition: transform ease-out 0.15s !important;
+  animation: i-amphtml-story-dev-tools-fade-in 0.5s ease-in-out !important;
+}
+
+@keyframes i-amphtml-story-dev-tools-fade-in {
```

Comment:
Should you use a delay instead? The 4th parameter of an `animation` or `transition` is the delay.
ie: `animation: name 0.3s curve 0.2s` will wait for `0.2s` to start

Reply:
Tried to use it at the beginning but didn't work as expected. If we use a delay, while waiting for that delay the opacity will be 1, then when the animation starts, it's set to 0 and animated back to 1. Definitely not what we want, so we might need to add `opacity: 0` (without `!important`) to the elements and then apply the animation with `fill-mode: forwards`. We'd need all that to be able to remove the 50% keyframe.

Tbh I think the current solution is simpler than that and avoids anti-patterns like not having the important, relying on fill-modes in animations, etc.

Reply:
Why is this delay needed in the first place? You're using a lot of delays to animate and they're all inter connected but living in different places, it will be next to impossible to change any animation timing.

Reply:
Because the devices start with `transform: translate3d(0, 0, 0)` so if you don't delay the fade-in, it fades in while it moves into position and that looks weird. I can disable the transition for newly created devices and add it after 1ms so it's properly positioned when the fade-in starts, sounds better?

Reply:
Edit: added comment on the CSS saying why the fade-in requires the delay. The numbers make more sense with the comment. Also my proposal on the previous comment wouldn't work because the devices would overlap while fading-in, which is not desirable (the new device will fade in in the position of the right-most device so we have to wait until that device moves to start the fade-in).